test/flycheck-test.el

```diff
@@ -3059,11 +3059,16 @@ See https://github.com/flycheck/flycheck/issues/531 and Emacs bug #19206"))
   (let ((flycheck-emacs-lisp-check-declare t))
     (flycheck-ert-should-syntax-check
      "language/emacs-lisp/check-declare-warnings.el" 'emacs-lisp-mode
-     (if (version< emacs-version "25")
-         '(0 nil warning "`this-function-is-not-declared' was defined in this-file-does-not-exist.el: file not found"
-             :checker emacs-lisp)
+     (cond
+      ((version< emacs-version "25")
+       '(0 nil warning "`this-function-is-not-declared' was defined in this-file-does-not-exist.el: file not found"
+           :checker emacs-lisp))
+      ((version< emacs-version "26")
        '(9 1 warning "`this-function-is-not-declared' was defined in this-file-does-not-exist.el: file not found"
-           :checker emacs-lisp)))))
+           :checker emacs-lisp))
+      (t
+       '(9 nil warning "`this-function-is-not-declared' was defined in this-file-does-not-exist.el: file not found"
+           :checker emacs-lisp))))))
```

Comment:
26 has less information that 25.1?

Reply:
Looks like it.  I was surprised by that as well.  Maybe an oversight in a larger rewrite of checkdoc?